scripts/Run_clang-tidy.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+#
+# 22 februar 2017
+#
+SRC="./src"
+#
+where=`pwd`
```

Comment:
Most shell scripts in the scripts dir assume that they are always executed from the root dir. 
I think that this is a bit unflexible.

A solution which I would prefer is to base all relative paths from the dir in which the script itself lies. 
Getting the script directory, unfortunately, is astoundingly non trivial (see [here](https://stackoverflow.com/questions/59895)), especially if systems that either use really old bash versions or don't have it a all need to be supported (bsd?, illumos?, QNX?, apple os). 

Do we actually want to support those systems? I know that one dev of illumos posted some issues here. There also seem to be a bunch of apple users using uncrustify.

I use
```sh
script_dir="$(dirname "$(readlink -f "$0")")"
```
which also is not perfect, but at least not bash dependent.

----------------

Just to be safe I would quote all variables that save paths.

----------------

The German words should be replaced.